resources/builder.inc.sh

```diff
@@ -1190,9 +1206,15 @@ builder_finish_action() {
   fi
 
   local scope="[$THIS_SCRIPT_IDENTIFIER] "
+  local matched_action="$action$target"
 
-  if [[ "$action$target" == "${_builder_current_action}" ]]; then
+  if [[ "$matched_action" == "${_builder_current_action}" ]]; then
     if [[ $result == success ]]; then
+      # Sanity check:  if there is a described output for this action, does the corresponding
+      # file or directory exist now?
+      if _builder_dep_output_defined $matched_action && ! _builder_dep_output_exists "$matched_action"; then
+        builder_warn "## $scope$action_name's described output does not exist"
```

Comment:
```suggestion
        builder_warn "## ${scope}Warning: $action_name was successful, but output does not exist."
        builder_warn "## ${scope}Expected output: '${_builder_dep_path[$matched_action]}'."
```

Giving the expected file might help clarify, especially if the described output is not a static string, e.g. when injecting configuration variables.